### PR TITLE
Adding documentation for Container Resolution

### PR DESCRIPTION
**Pull request by danielgsims**

### CODE REVIEW

**Comment by danielgsims** [comment]
This was brought up in: https://github.com/slimphp/Slim/issues/2080

File: docs/objects/router.md [hunk 1]
```diff
@@ -376,3 +378,27 @@ $app->get('/method1', '\MyController:method1');
...
+{% highlight php %}
+$container = new \Slim\Container
+$app = new \Slim\App($container);
+$container['home'] = function() {
+    return new HomeController;
+};
+$app->get('/index','home:index');
+{% endhighlight %}
+
+This allows you to leverage the Symfony Container for dependency injection. If you would prefer to
```
Comment (akrabat) [inline docs/objects/router.md]: It's not the Symfony container…

**Comment by danielgsims** [comment]
@akrabat - Sorry, I was thinking sensio for Pimple, but I'll make it non-specific for the container changes.

**Comment by akrabat** [comment]
Thanks.

**Pull request merged**